Source/WebCore/crypto/algorithms/CryptoAlgorithmHKDF.cpp

```diff
@@ -75,9 +75,9 @@ void CryptoAlgorithmHKDF::importKey(CryptoKeyFormat format, KeyData&& data, cons
     callback(CryptoKeyRaw::create(parameters.identifier, WTFMove(std::get<Vector<uint8_t>>(data)), usages));
 }
 
-ExceptionOr<size_t> CryptoAlgorithmHKDF::getKeyLength(const CryptoAlgorithmParameters&)
+ExceptionOr<std::optional<size_t>> CryptoAlgorithmHKDF::getKeyLength(const CryptoAlgorithmParameters&)
 {
-    return 0;
+    return std::optional<size_t>();
```

Comment:
Nit:  More idiomatic in WebKit to do this:

```suggestion
    return { };
```

Reply:
I tried that first. It does not compile. Exception or only has rvalue constructors.